File: 5_level/4258_lista_de_strings.py
def main():
    string = input("Digite uma String de números: \n")
    numbers = string.split(" ")
    n = len(numbers)
    
    out = "O inverso dos números digitados é: "

    for i in range(n - 1, -1, -1):
        out += numbers[i]
        if i == 0:
            out += "."
        else:
            out += ", "
    
    print(out)

main()

File: 5_level/4258_lista_de_strings.cpp
#include <bits/stdc++.h>

using namespace std;

int main ()
{
    vector <string> vector;
    string numbers;

    cout << "Digite uma String de números: \n";
    getline(cin, numbers);

    cout << "O inverso dos números digitados é: ";

    for (int i = vector.size() - 1; i >= 0; i--)
    {
        cout << vector[i];

        if (i == 0)
        {
            cout << ".\n";
        }
        else
        {
            cout << ", ";
        }
    }
    

    return 0;
}